ivy/functional/backends/jax/__init__.py

```diff
@@ -233,3 +233,12 @@ def closest_valid_dtype(type=None, /, as_native=False):
 from .experimental import *
 from . import control_flow_ops
 from .control_flow_ops import *
+
+
+# sub-backends
+try:
```

Comment:
Is there a reason to have try except statement here? we should have the `sub_backend` there by default, even if it's empty.

Reply:
mainly because i don't want people adding new backends to be forced to add the sub_backends folder and it's __init__ file, whenever a backend adds a sub_backend they can start adding folders/files/code needed